src/vs/base/common/htmlContent.ts

```diff
@@ -41,11 +41,11 @@ export class MarkdownString implements IMarkdownString {
 	get isTrusted() { return this._isTrusted; }
 	get supportThemeIcons() { return this._supportThemeIcons; }
 
-	appendText(value: string): MarkdownString {
+	appendText(value: string, newlineStyle: 'break' | 'paragraph' = 'paragraph'): MarkdownString {
```

Comment:
I would prefer to define a new enum for `newlineStyle`, but I don't actually own this file so I don't think I get the last say.